src/Stryker.Core/Stryker.Core/Initialisation/ProjectFileReader.cs

```diff
@@ -32,49 +32,56 @@ public ProjectFile ReadProjectFile(XDocument projectFileContents, string project
-                string referencesString = string.Join(", ", projectReferences);
-                if (string.IsNullOrEmpty(projectUnderTestNameFilter))
+                string referencesString = string.Join (", ", projectReferences);
+                if (string.IsNullOrEmpty (projectUnderTestNameFilter))
                 {
-                    throw new NotSupportedException("Only one referenced project is supported, please add the --project-file=[projectname] argument to specify the project to mutate", innerException: new Exception($"Found the following references: {referencesString}"));
+                    throw new NotSupportedException ("Only one referenced project is supported, please add the --project-file=[projectname] argument to specify the project to mutate", innerException : new Exception ($"Found the following references: {referencesString}"));
                 }
                 else
                 {
-                    var searchResult = projectReferences.Where(x => x.ToLower().Contains(projectUnderTestNameFilter.ToLower())).ToList();
-                    if(!searchResult.Any())
+                    var searchResult = projectReferences.Where (x => x.ToLower ().Contains (projectUnderTestNameFilter.ToLower ())).ToList ();
+                    if (!searchResult.Any ())
                     {
-                        throw new ArgumentException($"No project reference matched your --project-file={projectUnderTestNameFilter} argument to specify the project to mutate, was the name spelled correctly?", innerException: new Exception($"Found the following references: {referencesString}"));
-                    } else if (searchResult.Count() > 1)
+                        throw new ArgumentException ($"No project reference matched your --project-file={projectUnderTestNameFilter} argument to specify the project to mutate, was the name spelled correctly?", innerException : new Exception ($"Found the following references: {referencesString}"));
+                    }
+                    else if (searchResult.Count () > 1)
                     {
-                        throw new ArgumentException($"More than one project reference matched your --project-file={projectUnderTestNameFilter} argument to specify the project to mutate, please specify the name more detailed", innerException: new Exception($"Found the following references: {referencesString}"));
+                        throw new ArgumentException ($"More than one project reference matched your --project-file={projectUnderTestNameFilter} argument to specify the project to mutate, please specify the name more detailed", innerException : new Exception ($"Found the following references: {referencesString}"));
                     }
-                    return searchResult.Single();
+                    return searchResult.Single ();
                 }
             }
-            else if (!projectReferences.Any())
+            else if (!projectReferences.Any ())
```

Comment:
Something in your linter/formatter seems to have added a space behind almost every method call in this file. I would suggest you turn off this behaviour in your linter as it's contaminating the pull request.

Reply:
I apologize for the large number of issues. I have copied the code over, made my modifications again, then diffed the files. They seem to be identical save for the lines I added.

Reply:
No problem, we appreciate the work